staging/kos/pkg/networkfabric/interfaces.go

```diff
@@ -18,33 +18,116 @@ package networkfabric
 
 import "net"
 
-// Interface is the interface of a network fabric which allows
-// the user to implement Netowrk Interfaces. The implementer
-// MUST return nil when the user attempts to delete an Interface
-// which does not exist.
+// Interface is the contract of a VXLAN network fabric.
+// It declares functions to create Network Interfaces that are part of a VXLAN
+// segment.
+// The VXLAN segment of a Network Interface is an invocation argument of the
+// functions to create the Network Interface.
+// All traffic sent/received to/from the Network Interfaces created via an
+// implementer of this contract MUST be VXLAN-tunneled.
+//
+// This contract makes a distinction between local and remote Network Interfaces.
+//
+// A local Network Interface is the networking state of a guest that is bound to
+// the same node as the user of this contract. Creating a local Network
+// Interface means creating a Linux network device and configuring the node's
```

Comment:
I think the more familiar term in Linux is "network interface".  "device" suggests something found in `/dev`.

Reply:
e919b51 performs the suggested rewording. I do wonder if the new wording is still confusing though. Throughout the comments (and in function names) the hard state created by the fabric is called "local Network Interface". This leads to comments such as "a local Network Interface is implemented as a Linux network interface...".

Reply:
Yeah, the problem is that we need a technical term for our software artifacts that is not the same as "network interface" because one of our things is more than a network interface (it includes the flows) and, in some (remote) cases, less.  If we had one golang type, we could use that as the term.  But we have two (LocalNetIfc and RemoteNetIfc).

Reply:
I think it's ok to use the names of the Golang types, because the comments always make a distinction between "local Network Interface" and "remote Network Interface", hence there's no need for a catch-all name.